src/products/index.html

```diff
@@ -1,5 +1,5 @@
 <yield>
   <gb-list items="{ $products.products }" layout="grid">
-    <gb-product product="{ $item }"></gb-product>
+    <gb-product product="{ $item }" display="{ $products.variant.display }" field="{ $products.variant.field }"></gb-product>
```

Comment:
shouldn't need to pass these (`display` and `field`) here, instead they will just be determined by whoever is adding the variant capability to the `<gb-product>` container